### PR TITLE
add snp indel filter

### PR DESCRIPTION
**Pull request by mattsolo1**

### CODE REVIEW

File: utils.py [hunk 1]
```diff
@@ -75,6 +75,10 @@ def add_consequence_to_variant(variant):
     variant['HGVSc'] = get_transcript_hgvs(worst_csq)
     variant['HGVS'] = get_proper_hgvs(worst_csq)
     variant['CANONICAL'] = worst_csq['CANONICAL']
+    if len(variant['ref']) > 1 or len(variant['alt']) > 1:
+        variant['indel'] = True
```
Comment (konradjk) [inline utils.py]: How about:
```
variant['indel'] = len(variant['ref']) > 1 or len(variant['alt']) > 1
```

File: static/exac.js [hunk 2]
```diff
@@ -554,16 +554,81 @@ function draw_region_coverage(raw_data, metric, ref) {
...
+  ''
+]
+
+var categoryDefinitions = {
+  all: csq_order,
+  lof: csq_order.slice(0, csq_order.indexOf('frameshift_variant')),
+  missense: csq_order.slice(0, csq_order.indexOf('synonymous_variant')),
+}
+
+function uniqueMajorConsequences(variants_in_gene) {
```
Comment (bw2) [inline static/exac.js]: Is uniqueMajorConsequences used elsewhere?

File: templates/transcript_display.html [hunk 3]
```diff
@@ -422,7 +428,7 @@
         //Ellipse
         .append("ellipse")
         .attr("class", function(d) {
-            return "track_variant " + d.category;
+            return "track_variant " + d.major_consequence;
```
Comment (bw2) [inline templates/transcript_display.html]: might need css classes for each major_consequence (?)

File: static/exac.js [hunk 4]
```diff
@@ -554,16 +554,81 @@ function draw_region_coverage(raw_data, metric, ref) {
     }
 }
 
+var csq_order = [
```
Comment (bw2) [inline static/exac.js]: I like the idea of having these on the client side. 
We should double-check that VEP v85 hasn't added any new consequence terms.

File: templates/region_display.html [hunk 5]
```diff
@@ -140,7 +146,7 @@
         //Ellipse
         .append("ellipse")
         .attr("class", function(d) {
-            return "track_variant " + d.category;
+            return "track_variant " + d.major_consequence;
```
Comment (bw2) [inline templates/region_display.html]: might need css classes for each major_consequence (?

File: static/exac.js [hunk 6]
```diff
@@ -591,16 +591,13 @@ var csq_order = [
...
-  lof: csq_order.slice(0, csq_order.indexOf('frameshift_variant')),
-  missense: csq_order.slice(0, csq_order.indexOf('synonymous_variant')),
-}
-
-function uniqueMajorConsequences(variants_in_gene) {
-  return _.uniq(variants_in_gene.map(function(variant) {
-    return variant.major_consequence
-  }))
+  lof: csq_order.slice(0, csq_order.indexOf('frameshift_variant') + 1),
+  missense: csq_order.slice(0, csq_order.indexOf('missense_variant') + 1),
```
Comment (konradjk) [inline static/exac.js]: Shouldn't this start at `csq_order.indexOf('frameshift_variant') + 1` (missense should not include LoF unless you want to call it lof+missense, but then that might get a bit awkward).

Comment (mattsolo1) [inline static/exac.js]: how about:

```
var categoryDefinitions = {
  all: csq_order,
  lof: csq_order.slice(0, csq_order.indexOf('stop_lost')),
  missense: csq_order.slice(
      csq_order.indexOf('stop_lost'),
      csq_order.indexOf('protein_altering_variant')
  ),
}
categoryDefinitions.missenseAndLof =
    categoryDefinitions.lof.concat(categoryDefinitions.missense)
```
yields
```
{
  "all": [
    "transcript_ablation",
    "splice_acceptor_variant",
    "splice_donor_variant",
    "stop_gained",
    "frameshift_variant",
    "stop_lost",
    "start_lost",
    "inframe_insertion",
    "inframe_deletion",
    "missense_variant",
    "protein_altering_variant",
    "incomplete_terminal_codon_variant",
    "stop_retained_variant",
    "synonymous_variant",
    "coding_sequence_variant",
    "mature_miRNA_variant",
    "5_prime_UTR_variant",
    "3_prime_UTR_variant",
    "non_coding_transcript_exon_variant",
    "non_coding_exon_variant",
    "NMD_transcript_variant",
    "non_coding_transcript_variant",
    "nc_transcript_variant",
    "downstream_gene_variant",
    "TFBS_ablation",
    "TFBS_amplification",
    "TF_binding_site_variant",
    "regulatory_region_ablation",
    "regulatory_region_amplification",
    "feature_elongation",
    "regulatory_region_variant",
    "feature_truncation",
    "intergenic_variant",
    ""
  ],
  "lof": [
    "transcript_ablation",
    "splice_acceptor_variant",
    "splice_donor_variant",
    "stop_gained",
    "frameshift_variant"
  ],
  "missense": [
    "stop_lost",
    "start_lost",
    "inframe_insertion",
    "inframe_deletion",
    "missense_variant"
  ],
  "missenseAndLof": [
    "transcript_ablation",
    "splice_acceptor_variant",
    "splice_donor_variant",
    "stop_gained",
    "frameshift_variant",
    "stop_lost",
    "start_lost",
    "inframe_insertion",
    "inframe_deletion",
    "missense_variant"
  ]
}
```

**Pull request merged**